### PR TITLE
PXB-2897 Release notes and release note tasks for PXB-8.0.30-23

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file: release-notes/8.0/8.0.30-23.0.md
modified:   mkdocs-base.yml
modified:   docs/release-notes.md

### CODE REVIEW

File: docs/release-notes/8.0/8.0.30-23.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+|---|---|
+| Install instructions | [Install Percona XtraBackup](https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html) |
+
+[Percona XtraBackup for MySQL Databases](https://www.percona.com/software/mysql-database/percona-xtrabackup) enables MySQL backups without blocking user queries. Percona XtraBackup is ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, Percona XtraBackup drives down backup costs while providing unique features for MySQL backups. [Commercial support contracts are available](https://www.percona.com/services/support/mysql-support/).
+
+## Release highlights
+
+New features and improvements introduced in Percona XtraBackup 8.0.30-23:
+
+* Percona XtraBackup adds support for `Zstandard (ZSTD)` compression algorithm. `ZSTD` is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios.
```
Comment (patrickbirch) [inline docs/release-notes/8.0/8.0.30-23.0.md]: add a link to the zstandard document

File: docs/release-notes/8.0/8.0.30-23.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Release highlights
+
+New features and improvements introduced in Percona XtraBackup 8.0.30-23:
+
+* Percona XtraBackup adds support for `Zstandard (ZSTD)` compression algorithm. `ZSTD` is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios. 
+
+* Percona XtraBackup implements the `Smart memory estimation` feature to compute the memory required to `--prepare` a backup. Percona XtraBackup has extended the crash recovery logic to extract the formula used to allocate memory. Now, in the backup phase, while copying redo log entries, Percona XtraBackup computes the required memory for `--prepare`. Percona XtraBackup also takes into consideration the number of InnoDB pages to be fetched from the disk. Percona XtraBackup checks the server's available free memory and uses that memory up to the limit specified in the `--use-free-memory-pct` parameter to run `--prepare`.
```
Comment (patrickbirch) [inline docs/release-notes/8.0/8.0.30-23.0.md]: add a link to the smart memory estimation document

File: docs/release-notes/8.0/8.0.30-23.0.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.30-23.0.md]: Added.

File: docs/release-notes/8.0/8.0.30-23.0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+## Improvements
+
+[PXB-2710](https://jira.percona.com/browse/PXB-2710): Implements support for `Smart Memory Estimation`.
+[PXB-2844](https://jira.percona.com/browse/PXB-2844): Adjusts the logic to detect multi-threaded slave.
+
+## Bugs fixes
+
+[PXB-2681](https://jira.percona.com/browse/PXB-2681): Percona XtraBackup got stuck during incremental backup prepare of encrypted tables.
+[PXB-2517](https://jira.percona.com/browse/PXB-2517): Using innobackupex for database backup failed.
+[PXB-2702](https://jira.percona.com/browse/PXB-2702): Backup failed when undo truncation log is present.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.30-23.0.md]: Add a note:
Thank you Dmitry Smal for your contribution for initial patch.

File: docs/release-notes/8.0/8.0.30-23.0.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## Improvements
+
+[PXB-2710](https://jira.percona.com/browse/PXB-2710): Implements support for `Smart Memory Estimation`.
+[PXB-2844](https://jira.percona.com/browse/PXB-2844): Adjusts the logic to detect multi-threaded slave.
+
+## Bugs fixes
+
+[PXB-2681](https://jira.percona.com/browse/PXB-2681): Percona XtraBackup got stuck during incremental backup prepare of encrypted tables.
+[PXB-2517](https://jira.percona.com/browse/PXB-2517): Using innobackupex for database backup failed.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.30-23.0.md]:
```suggestion
[PXB-2517](https://jira.percona.com/browse/PXB-2517): Check if files-from is successfully closed before rsync (Thanks to Garen Chan for the patch).
```

File: docs/release-notes/8.0/8.0.30-23.0.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+[PXB-2702](https://jira.percona.com/browse/PXB-2702): Backup failed when undo truncation log is present.
+[PXB-2729](https://jira.percona.com/browse/PXB-2729): Percona XtraBackup compilation failed if GTest packages are installed.
+[PXB-2810](https://jira.percona.com/browse/PXB-2810): Percona XtraBackup crashed when AIO initialization failed.
+[PXB-2840](https://jira.percona.com/browse/PXB-2840): Xbcloud attempted to create a non-existing bucket. 
+
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.30-23.0.md:68]: Add:
 - PXB-2874
 - PXB-2809
 - PXB-2844
 - PXB-2810 (Thanks to Yan Huang for the patch.)
 -

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.30-23.0.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.30-23.0.md]: added

File: docs/release-notes/8.0/8.0.30-23.0.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.30-23.0.md]: Updated

File: docs/release-notes/8.0/8.0.30-23.0.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.30-23.0.md:68]: Updated

File: docs/release-notes/8.0/8.0.30-23.0.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.30-23.0.md]: Updated

**Review by altmannmarcelo — APPROVED** [review]
LGTM

File: docs/release-notes/8.0/8.0.30-23.0.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | October 2022 |
+|---|---|
+| Install instructions | [Install Percona XtraBackup](https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html) |
+
```
Comment (rahulmalik87) [inline docs/release-notes/8.0/8.0.30-23.0.md:24]: Should we mention somewhere that the page-tracking for an incremental backup feature is now GA and provide a documentation link to it

**Review by patrickbirch — APPROVED** [review]
Good
Do a squash merge

File: docs/release-notes/8.0/8.0.30-23.0.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.30-23.0.md:24]: Added. Thank you.

**Review by george-lorch — APPROVED** [review]

**Pull request merged**